spec/definitions/Scheduling/Time.yaml

```diff
@@ -0,0 +1,3 @@
+type: string
+format: date-time
```

Comment:
I don't think you would supply the format and a pattern. I think you only want the time, so you should remove `date-time`.